README_DORA.md

```diff
@@ -0,0 +1,11 @@
+# Dora Fork of /topic0 
+
+## Important files
+
+The file `DORA_CHAIN_LIST` contains a list of dora-supported chainIDs, which are then used to build the signature database. This should be updated if one wants to rebuild with a different sign
```

Comment:
feel like this sentence is supposed to end differently, "different set of chains" or similar ?

Reply:
fixed now, good to merge?